_posts/2018-04-06-all-hands.md

```diff
@@ -0,0 +1,183 @@
+  - Stabilize a forward-compatible subset of procedural macros that explicitly
+    opts out of hygiene (by asking all names to be interpreted at the call
+    site).
+  - Work out a revised API surface for procedural macros based on what we've learned so far.
+  - Stabilize importing macros through normal `use` statements.
+  - **Alex Crichton will write an RFC for the above**.
+- **Extern types**: worked through [remaining issues for stabilization](https://github.com/rust-lang/rust/issues/43467#issuecomment-377521693).
+- **Improvements to `derive`**: a [proposal](https://github.com/rust-lang/rfcs/pull/2385) to make `derive` more ergonomic in Rust 2018.
+- **Best practices**: set up a cross-cutting guidelines effort, encompassing the
+  API guidelines but also including style, lints, and Lang Team recommendations.
+
+### Libraries
+
+- **SIMD**: talked through final steps of stabilization; we hope to [stabilize in 1.27](https://github.com/rust-lang/rust/issues/48556#issuecomment-378184312).
+- **Allocators**: developed a conservative proposal for stabilizing global allocators; **Simon Sapin will post the proposal on the tracking issue**.
+
+### Compiler
+
+- **Tool integration**: extensive discussion and planning about the needs of
+  compiler clients, composable plugins, and the compiler's new query
+  architecture.
+- **Query compilation**: a plan for end-to-end query compilation, i.e. fully-incremental compilation.
+- **libsyntax**: a long-run plan for a revamped libsyntax, shared amongst a variety of tools.
+- **Contribution**: improved contribution experience for the compiler.
+
+### Community
+
+- **Mozilla Activate**: shipped a revamp of the page
+- **RustBridge "in a box"**: shipped a ton of resources.
+- **Events**: developed event standards, including:
+  - Diversity Outreach
+  - [Speaker support and quality](https://github.com/rust-community/events-team/pull/18)
+  - [Program guidelines](https://github.com/rust-community/events-team/pull/19)
+  - [Review committee guidelines](https://github.com/rust-community/events-team/pull/20)
+  - Communication and transparency (e.g. [timelines](https://github.com/rust-community/events-team/pull/22))
+  - Attendee experience, accessibility and support
+
+### Documentation
+
+- **Edition planning**: determined resources needed for the 2018 edition, what
+  that means for the Rust Bookshelf, and who will be responsible for each.
+- **Team blog**: “This week in Rust docs” is going to move to a new [docs team blog](https://rust-docs.github.io/blog/)!
+- **Doxidize** (aka rustdoc2): made [initial public release](https://github.com/steveklabnik/doxidize); it's like https://docusaurus.io/ but for Rust.
+- **Intermediate-level docs**: contributed to [idea generation](https://github.com/rust-docs/team/issues/8).
+
+### Tools
+
+- **Edition planning**, especially for the rustfix tool.
+- **Clippy lint audit**: developed plan for reaching 1.0 on Clippy this year.
```

Comment:
Should we mention here that we grouped all lints into one of the following?

Correctness, performance, style, complexity, pedantic and nursery